formula-coroutines/src/main/java/com/instacart/formula/coroutines/FlowRuntime.kt

```diff
@@ -0,0 +1,43 @@
+package com.instacart.formula.coroutines
+
+import com.instacart.formula.FormulaRuntime
+import com.instacart.formula.IFormula
+import com.instacart.formula.internal.ThreadChecker
+import kotlinx.coroutines.channels.awaitClose
+import kotlinx.coroutines.channels.trySendBlocking
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.callbackFlow
+import kotlinx.coroutines.flow.distinctUntilChanged
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+
+object FlowRuntime {
```

Comment:
Do we want to add `@ExperimentalCoroutinesApi` here or opt in at the module level?

Reply:
Good question - I will mark this method with this annotation so that the consumers are aware when using the API.